navigation/administration_page.md

```diff
@@ -4,7 +4,7 @@ There are four ways to configure pipelines etc.
 
 1.  Via the Admin UI described below
 2.  Direct XML edit via the Admin UI's Config XML tab
-3.  Some limited configuration is possible via [Config API](../api/configuration_api.md)
+3.  Some limited configuration is possible via [Config API](https://api.gocd.org/current/#pipeline-config)
```

Comment:
I think we have a good number of apis to configure a lot of things on Go. How about removing the word limited?